1/Vector_3d.cpp

```diff
@@ -0,0 +1,91 @@
+#include "Vector_3d.h"
+#include <limits>
+
+Vector_3d::Vector_3d(double a, double b, double c)
+{
+	this->a = a;
+	this->b = b;
+	this->c = c;
+}
+Vector_3d::Vector_3d()
+{
+	this->a = 0;
+	this->b = 0;
+	this->c = 0;
+}
+
+Vector_3d Vector_3d::operator+(const Vector_3d& other)
+{
+	return Vector_3d(this->a + other.a, this->b + other.b, this->c + other.c);
+}
+
+Vector_3d Vector_3d::operator-(const Vector_3d& other)
+{
+	return Vector_3d(this->a - other.a, this->b - other.b, this->c - other.c);
+}
+
+double Vector_3d::get_scalar_composition(const Vector_3d& other)
+{
+	return this->a * other.a + this->b * other.b + this->c * other.c;
+}
+Vector_3d Vector_3d::get_multiplication_by_number(const Vector_3d, double scalar)
+{
+	return Vector_3d(this->a * scalar, this->b * scalar, this->c * scalar);
+}
+
+bool Vector_3d::are_qual(const Vector_3d& other)
+{
+	if (std::fabs(this->a - other.a) < std::numeric_limits<double>::epsilon() && std::fabs(this->b - other.b) < std::numeric_limits<double>::epsilon() &&
+		std::fabs(this->c - other.c) < std::numeric_limits<double>::epsilon())
+	{
+		return true;
+	}
+	else return false;
+}
```

Comment:
`return abs((this->a - other.a) <= std::numeric_limits<double>::epsilon() && ...;`